root/etc/e-smith/templates/httpd/vhost-extra/25IndexesOptions

```diff
@@ -0,0 +1,14 @@
+{
+use esmith::ConfigDB;
+my $vdb = esmith::ConfigDB->open_ro('vhosts') || die("Can't open vhosts db");
+
+my $indexes = $vdb->get_prop("$VhostName",'Indexes') || '';
+return unless ($indexes eq 'enabled');
+
+$OUT .= << 'EOF';
+        #25IndexesOption
+        <LocationMatch "^/">
```

Comment:
I guess yes, but of course before to deploy it....a bit of tests is needed.

Even with the indexes option when you put an index.* In the vhost folder, it is displayed....but I don't know for subfolder of a webapp.
For security the indexes property is set to disabled of course. If it is enabled it is by the sysadmin. 
Concerning the indexes property I removed in the vhost template.....we can let it here if you prefer....it is not usable du to the welcome.conf